server/liveblog/syndication/syndication.py

```diff
@@ -94,6 +97,12 @@ def send_syndication_post(self, post, action='created'):
             logger.warning('Not sending post "{}": syndicated content.'.format(post['_id']))
             return
 
+        items = extract_post_items_data(post)
```

Comment:
If you find that you're checking for these post types in more than one place, you should probably put this logic in utils in a function which takes a post as a parameter, or simply have extract_post_data instead of extract_post_items_data.